components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -1440,21 +1440,24 @@ def setDefaultGroup(self, group_id, exp_id=None):
         admin_serv.setDefaultGroup(
             ExperimenterI(exp_id, False), ExperimenterGroupI(group_id, False))
 
-    def updatePermissions(self, obj, permissions):
+    def updatePermissions(self, group, permissions):
         """
-        Allow to change the permission on the object.
+        Allow to change the permission of the group.
 
-        @param obj      A wrapped entity or an unloaded reference to an
+        @param group      A wrapped entity or an unloaded reference to an
                         entity. Not null.
-        @type obj       BlitzObjectWrapper
+        @type group       BlitzObjectWrapper
         @param perm     The permissions value for this entity. Not null.
         @type perm      PermissionsI
         """
 
-        admin_serv = self.getAdminService()
-        if permissions is not None:
-            logger.warning("WARNING: changePermissions was called!!!")
-            admin_serv.changePermissions(obj._obj, permissions)
+        perms = str(permissions)
+        logger.debug("Chmod of group ID: %s to %s" % (group.id, perms))
+        command = Chmod(type="/ExperimenterGroup",
+                        id=group.id,
+                        permissions=perms)
+        cb = self.c.submit(command)
```

Comment:
No. This is precisely what `c.submit` does. It's very different from `c.sf.submit`. (Sorry for the poor naming choice. My fault!)